atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/NoOpTransactionScopedCache.java

```diff
@@ -43,11 +43,22 @@ public void delete(TableReference tableReference, Set<Cell> cells) {}
     @Override
     public Map<Cell, byte[]> get(
             TableReference tableReference,
-            Set<Cell> cell,
+            Set<Cell> cells,
             BiFunction<TableReference, Set<Cell>, ListenableFuture<Map<Cell, byte[]>>> valueLoader) {
-        return AtlasFutures.getUnchecked(valueLoader.apply(tableReference, cell));
+        return AtlasFutures.getUnchecked(getAsync(tableReference, cells, valueLoader));
     }
 
+    @Override
+    public ListenableFuture<Map<Cell, byte[]>> getAsync(
+            TableReference tableReference,
+            Set<Cell> cells,
+            BiFunction<TableReference, Set<Cell>, ListenableFuture<Map<Cell, byte[]>>> valueLoader) {
+        return valueLoader.apply(tableReference, cells);
+    }
+
+    @Override
+    public void close() {}
```

Comment:
We _could_ make this throw like the other caches, but I don't think we need to (since it doesn't actually affect semantics if it does or not as nothing is being cached anyway.